nailgun/entities.py

```diff
@@ -1116,6 +1116,93 @@ def read(self, entity=None, attrs=None, ignore=None, params=None):
             entity, attrs, ignore, params)
 
 
+class ExternalUserGroup(
+        Entity,
+        EntityCreateMixin,
+        EntityDeleteMixin,
+        EntityReadMixin):
+    """A representation of a External Usergroup entity.
+
+   ``usergroups`` must be passed in when this entity is instantiated.
+
+   :raises: ``TypeError`` if ``usergroups`` is not passed in.
+
+    # Create external usergroup
+    ExternalUserGroup(name='foobargroup',usergroups=usergroup,auth_source=auth).create()
+    # Read external usergroup
+    ExternalUserGroup(id=<id>, usergroups=usergroup).read()
+    # Delete external usergroup
+    ExternalUserGroup(id=<id>, usergroups=usergroup).delete()
+    # Refresh external usergroup
+    ExternalUserGroup(id=<id>, usergroups=usergroup).refresh()
+    """
+    def __init__(self, server_config=None, **kwargs):
+        _check_for_value('usergroups', kwargs)
+        self._fields = {
+            'name': entity_fields.StringField(required=True),
+            'usergroups': entity_fields.OneToOneField(
```

Comment:
If it's one to one field (only one user group should be passed) - then pls rename it to `'usergroup'` accordingly.